pandamium_datapack/data/pandamium/functions/main_loop.mcfunction

```diff
@@ -58,7 +58,11 @@ function pandamium:misc/auto_item_timer
 
 function pandamium:misc/map_specific/loop
 
-execute as @a[gamemode=!spectator,scores={active_particles=1..}] at @s run function pandamium:misc/particles/main
+effect clear @a[scores={hidden=1}]
+effect give @a[scores={hidden=1}] invisibility 1000000 0 true
```

Comment:
it might be cleaner to do the effects similar to the spawn effects (constantly adding a very short effect)? that way the invisibility should disappear automatically once the player unhides